packages/dd-trace/test/appsec/blocking.spec.js

```diff
@@ -58,20 +58,22 @@ describe('blocking', () => {
 
     it('should log warn and not send blocking response when headers have already been sent', () => {
       res.headersSent = true
-      block(req, res, rootSpan)
+      const blocked = block(req, res, rootSpan)
 
+      expect(blocked).to.be.false
       expect(log.warn).to.have.been
         .calledOnceWithExactly('[ASM] Cannot send blocking response when headers have already been sent')
-      expect(rootSpan.addTags).to.not.have.been.called
+      expect(rootSpan.setTag).to.have.been.called
```

Comment:
called with what ? you never check that it's been called with the block failed tag ?

Reply:
```suggestion
      expect(rootSpan.setTag).to.have.been.calledOnceWithExactly('_dd.appsec.block.failed', 1)
```